docs/raffle-queue.md

```diff
@@ -0,0 +1,28 @@
+# CannonKeys Raffle Queue How-To
+##### Link: https://hype.cannonkeys.com/
+&nbsp;
+If you arrive a few minutes before the drop begins, you should see a screen like this:
+
+![](images/raffle_queue/before_start.png)
+
+At the exact time the drop starts, you can either refresh your browser or click the refresh button in the middle. Keep in mind that if you click the refresh button, you will need to wait a few seconds until you click again.
+
+If the load went smoothly, you should see something like the picture below. If not, you may need to try refreshing again.
+
+![](images/raffle_queue/join_queue.png)
+
+Click the “join” button and you should be put into queue. If you don’t see text below the loading bar, you are not in queue. Try refreshing and clicking “join” again.
+
+![](images/raffle_queue/in_queue.png)
+
+If you are lucky you may be brought into a screen to select your options!
+
+![](images/raffle_queue/choose_options.png)
+
+#### Please keep in mind that even if you make it past the queue, you are not guaranteed your item, make sure to choose options and checkout as fast as you can!
```

Comment:
These are showing up as navigation in table of contents ATM.
![Screen Shot 2021-02-22 at 12 29 19 PM](https://user-images.githubusercontent.com/25588015/108745839-9a5eed80-7509-11eb-9ef2-2e46bd9bc87e.png)


Can we add ## headers to set-up table of contents?
And maybe change these to alerts similar to the blue pill guide?

This is the syntax:

```
!!! Note
    This section assumes that you've purchased a Blue Pill as part of a kit from me, or from my shop. Most blue pills come with extra jumpers soldered. If this is the case for your Blue Pill, please visit [this page](bluepill-mod.md) for detailed instructions on how to modify it.
```